### ISSUE TITLE
Unable to present deviceChooser in custom view controller

### ISSUE THREAD

**Issue by bhagyas** [googlecast/CastVideos-ios#76]
SDK provides no way to specify the parent view controller for presenting the deviceChooser.

**Comment by anjaneesh** [comment]
Hello, do you mean, you want to control the wording that shows on the parent view while presenting the deviceChooser?

**Comment by bhagyas** [comment]
No, I would like to present the deviceChoose on a custom view controllers where the window is not directly accessible.
